docs/src/main/asciidoc/_extending-wildfly/Key_Interfaces_and_Classes_Relevant_to_Extension_Developers.adoc

```diff
@@ -67,9 +67,8 @@ interface) relate to the notion of managed resources in the AS.
 
 Each subsystem is responsible for managing one or more management
 resources. The conceptual characteristics of a management resource are
-covered in some detail in the link:Admin_Guide{outfilesuffix}#Management_resources[Admin
-Guide]; here we'll just summarize the main points. A management resource
-has
+covered in some detail in the <<management-resources,Admin Guide>>;
```

Comment:
This link works fine for me with the existing doc, but is broken with this change. This "Extending WildFly" doc is a different page from "Admin Guide" so an internal link will not work.

Reply:
Thanks for the review @bstansberry , yes, somehow I did not test correctly this link. Now it is fixed. 

Notice that the intention of this link is to jump to "2.3. Management resources" section on "WildFly Admin Guide".  In our current version, the link jumps to "WildFly Admin Guide" but it doesn't move to "2.3. Management resources". It is a very subtle difference.